### PR TITLE
Adds `scaffoldMessenger`

### PR DESCRIPTION
**Pull request by matuella**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ dependencies:
   equatable: ^2.0.0
   path: ^1.8.0
   path_provider: ^2.0.0
+  url_launcher: ^6.0.3
```
Comment (ggirotto) [inline pubspec.yaml:24]: Why this package was added on this PR? It's not used in any of the introduced files. Also, this package advises some extra setup in iOS to work properly as far as I understood: https://pub.dev/packages/url_launcher#installation

**Review by ggirotto — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (ggirotto) [reply pubspec.yaml:24]: I found its usage in this future PR: https://github.com/olmps/memo/pull/69/files but it doesn't have the iOS configuration. that the package describes in the tutorial. Are these steps optional?

Comment (matuella) [reply pubspec.yaml:24]: yeah, it should've been added in #69

Not sure about this config, on emulator it worked fine. Sadly, can't currently test on a physical device atm, but will keep this in mind before merging all these multi-PR changes

**Review by ggirotto — APPROVED** [review]

**Comment by matuella** [comment]
All updates made and merged in the last PR (#76) in this nested chain of collection execution PRs

**Pull request closed without merging**